ROMFS/px4fmu_common/init.d/rc.mc_apps

```diff
@@ -8,7 +8,7 @@
 # INAV, which defaults to 0 now. 
 if param compare INAV_ENABLED 1
 then
-	attitude_estimator_ekf start
+	attitude_estimator_q start
```

Comment:
Should we make this conditional based on a param while we're still actively developing this filter (e.g. gyro failover)? That would allow people to help testing it. I would love to merge the current flight tested state to master to prevent drift of the branches.